doc/vim-go.txt

```diff
@@ -629,6 +668,19 @@ if			      "inside a function", select contents of a function,
 
 
 ===============================================================================
+FUNCTIONS                                                       *go-functions*
+
+There is only one function at the moment and it is neovim only.
```

Comment:
I'm just going over it. No worries, take your time ;) I've went over the same bullshit back in university, that's life :)